src/Svg.Model/SvgExtensions.Pathing.cs

```diff
@@ -407,7 +411,11 @@ internal static System.Drawing.PointF ToAbsolute(System.Drawing.PointF point, bo
                     }
                     isEndFigure = false;
                     haveFigure = false;
-                    skPath.Close();
+                    if (!skPath.IsEmpty)
+                    {
+                        skPath.Close();
```

Comment:
Down stream code does no like a path that may start with a Close which can happen with this solution.